woocommerce/woocommerce-paypal-payments
# New Settings UI: Update onboarding button styles (3940)

### Description

Update the onboarding primary button styles to align with the new Figma design.

### Steps to Test

1. Enable the new Settings UI
2. Start the PayPal onboarding process
3. Verify that the design of the buttons aligns with the new [Figma design.](https://www.figma.com/design/uaM2fX0CAAQDgrxcvgTUep/Woo-Paypal-Plugin-High-Fidelity-Wireframes?node-id=5121-47350&t=sLsC12Mgp6N593F4-4)

|Before|After|
|-|-|
|<img width="936" alt="dashboard" src="https://github.com/user-attachments/assets/35a406d8-d28d-43ea-9d69-f1fa1602722d">|<img width="935" alt="overview" src="https://github.com/user-attachments/assets/557469d7-51be-4246-9fbf-f68a1c3e45bb">|
|<img width="936" alt="dashboard" src="https://github.com/user-attachments/assets/2ae86e77-ebe2-4723-8970-95b254c4c754">|<img width="935" alt="overview" src="https://github.com/user-attachments/assets/8a1e002e-d302-41eb-9883-f04920c54b7e">|
